common/shlibs

```diff
@@ -3860,3 +3860,6 @@ libtagparser.so.9 tagparser-9.1.2_1
 libqtutilities.so.6 qtutilities-6.0.4_1
 libkdsoap.so.1.9.0 KDSoap-1.9.0_1
 libstoken.so.1 stoken-0.92_1
+libjcat.so.1 libjcat-0.1.2_1
+libfwupd.so.2 fwupd-1.4.1_1
```

Comment:
These probably aren't needed since I think they're only used in the same package

Reply:
Removed.